phoenix-core/src/test/java/org/apache/phoenix/util/ScanUtilTest.java

```diff
@@ -515,9 +522,15 @@ public void testPhoenixTTLUtilMethods() throws SQLException {
                 KeyValue.Type type43 = KeyValue.Type.Put;
                 String value43 = "test.value.43";
                 long seqId43 = 1043L;
-                Cell cell43 = CellUtil.createCell(Bytes.toBytes(row),
-                        emptyColumnFamilyName, Bytes.toBytes(columnName),
-                        timestamp43, type43.getCode(), Bytes.toBytes(value43), seqId43);
+                Cell cell43 = ExtendedCellBuilderFactory.create(CellBuilderType.DEEP_COPY)
+                        .setRow(Bytes.toBytes(row))
+                        .setFamily(emptyColumnFamilyName)
+                        .setQualifier(Bytes.toBytes(columnName))
+                        .setTimestamp(timestamp43)
```

Comment:
These two in this file do need the ExtendedCellBuilderFactory because of the sequence id. Outside of the scope of this PR, but in the future we might want to figure out an IA.LimitedPrivate/Public way to do this, if possible.